Assets/Code/POLIMIgameCollective/EVENTM~1/EventManager.cs

```diff
@@ -0,0 +1,217 @@
+﻿using UnityEngine;
```

Comment:
I think that by mistake the folder has a weird name. Can you please change the "EVENTM-1" name? Thanks :)

Reply:
On it